src/pseudopeople/configuration/noise_configuration.py

```diff
@@ -96,3 +96,10 @@ def get_token_probability(
             dataset, noise_type, parameter_name="token_probability", column_name=column_name
         )
         return value
+
+    def has_row_noise_type(self, dataset_name: str, noise_type: str) -> bool:
```

Comment:
This may end up being better as, but that will be clearer when you implement column noise.
```suggestion
    def has_noise_type(self, dataset_name: str, noise_type: str, column: str | None = None) -> bool:
```

Reply:
I'll update the function when working on column noise but can update the function signature now.